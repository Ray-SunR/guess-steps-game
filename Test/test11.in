game 40 7 stdin stdin
3
10
7
8
9
10
12
game 40 7 stdin test11A1.txt
10
8
10
game 40 7 test11A1.txt stdin
10
8
10
quit
