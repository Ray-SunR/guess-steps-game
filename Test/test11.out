Total is 40, max deduction is 7
Player A's move
Total is 37
Player B's move
Total is 30
Player A's move
Total is 23
Player B's move
Total is 16
Player A's move
Total is 9
Player B's move
Total is 2
Player A's move
Total is -5
Player A wins
Score is
A 1
B 0
Total is 40, max deduction is 7
Player B's move
Total is 37
Player A's move
Total is 30
Player B's move
Total is 23
Player A's move
Total is 16
Player B's move
Total is 9
Player A's move
Total is 2
Player B's move
Total is -5
Player B wins
Score is
A 1
B 1
Total is 40, max deduction is 7
Player A's move
Total is 37
Player B's move
Total is 30
Player A's move
Total is 23
Player B's move
Total is 16
Player A's move
Total is 9
Player B's move
Total is 2
Player A's move
Total is -5
Player A wins
Score is
A 2
B 1
